composer.json

```diff
@@ -19,15 +19,16 @@
 		"bin/yaml-sort-checker"
 	],
 	"require": {
-		"php": "~7.2",
+		"php": "~7.2 | ^8.0",
 		"symfony/console": "~3.4|~4.3|~5.0",
 		"symfony/yaml": "~3.4|~4.3|~5.0"
 	},
 	"require-dev": {
-		"consistence/coding-standard": "3.9",
-		"jakub-onderka/php-parallel-lint": "1.0.0",
-		"phpstan/phpstan-shim": "0.11.19",
-		"phpunit/phpunit": "8.4.3"
+		"consistence/coding-standard": "^3.9",
+		"php-parallel-lint/php-parallel-lint": "^1.0.0",
```

Comment:
I usually lock dev dependencies to specific version to prevent surprises. There is no need for wide compatibility.

Reply:
But its adds unnecessary effort with updating the packages. But changed as your wish.